web/html/src/manager/admin/setup/products/products-scc-dialog.tsx

```diff
@@ -0,0 +1,200 @@
+import * as React from "react";
+import Network from "utils/network";
+import { Messages, MessageType } from "components/messages";
+import { Button } from "components/buttons";
+
+const msgMap = {
```

Comment:
This was originally missing in the source file, we had a discussion about this with Dario and our best guess is that it simply got lost over time in refactoring. My guess is that no one noticed because the network error branch was hit rarely enough or never.